src/test/java/teammates/test/cases/ui/InstructorFeedbackEditCopyPageActionTest.java

```diff
@@ -0,0 +1,57 @@
+package teammates.test.cases.ui;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertFalse;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.InstructorFeedbackEditCopyPageAction;
+import teammates.ui.controller.InstructorFeedbackEditCopyPageData;
+
+
+public class InstructorFeedbackEditCopyPageActionTest extends
+        BaseActionTest {
+    private static DataBundle dataBundle;
+    
+    @BeforeClass
+    public static void classSetUp() throws Exception {
+        printTestClassHeader();
+        dataBundle = loadDataBundle("/InstructorFeedbackEditCopyTest.json");
+        removeAndRestoreDatastoreFromJson("/InstructorFeedbackEditCopyTest.json");
+        
+        uri = Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_COPY_PAGE;
+    }
+    
+    @Test
+    public void testExecuteAndPostProcess() throws Exception{
+        InstructorAttributes instructor = dataBundle.instructors.get("teammates.test.instructor2");
+        String instructorId = instructor.googleId;
+        
+        gaeSimulation.loginAsInstructor(instructorId);
+        
+        
+        ______TS("Successful case");
+        
+        InstructorFeedbackEditCopyPageAction a = getAction();
+        AjaxResult r = getAjaxResult(a);
+        
+        assertFalse(r.isError);
+
+        InstructorFeedbackEditCopyPageData pageData = 
+                (InstructorFeedbackEditCopyPageData) r.data;
+        assertEquals(4, pageData.courses.size());
```

Comment:
The test data contains a course which is archived and a course that the instructor does not have permissions to create sessions in. I've added checks that these courses are not included below.